dmoj/problem.py

```diff
@@ -33,6 +34,8 @@ def __init__(self, problem_id, time_limit, memory_limit, load_pretests_only=Fals
             doc = yaml.safe_load(self.problem_data['init.yml'])
             if not doc:
                 raise InvalidInitException('I find your lack of content disturbing.')
+            init_check = Core(source_file=doc, schema_files=["judge/dmoj/problem_schema.yml"])
```

Comment:
Why don't you do `os.path.join(os.path.dirname(__file__), 'problem_schema.yml)` like a normal person? This only works when run from a certain directory...